Test/dafny0/ForbidNondeterminism.dfy

```diff
@@ -1,4 +1,4 @@
-// RUN: %baredafny verify %args --strict-definite-assignment --enforce-determinism "%s" > "%t" || true
+// RUN: %baredafny verify %args --strict-definite-assignment "%s" > "%t" || true
```

Comment:
Why this change? Wasn't that the point of the test? :)

Reply:
`--enforce-determinism` was testing that this flag has no effect when only verifying. This PR removing the option `--enforce-determinism` from the `verify` command so it's not possible to keep it here.